rulesets/standard.json

```diff
@@ -57,6 +57,14 @@
             ]
         }
     },
+    "//planned-disbursement/provider-org | //planned-disbursement/receiver-org | //transaction/provider-org | //transaction/receiver-org | //recipient-org-budget/recipient-org": {
+        "regex_matches": {
+            "cases": [
+                { "regex": "^(?!\s*$).+",
```

Comment:
It appears that this is not a valid JSON string:

```
>>> json.loads(r'{ "regex": "^(?!\s*$).+"}')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib/python2.7/json/__init__.py", line 326, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Invalid \escape: line 1 column 16 (char 16)
```

Reply:
It appears the fix to this issue is to only reference a subset of regex syntax described here: https://spacetelescope.github.io/understanding-json-schema/reference/regular_expressions.html#regular-expressions